EmbeddedScripts.Shared/Container.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace EmbeddedScripts.Shared
+{
+    public class Container
+    {
+        public void Register<T>(T instance, string alias)
```

Comment:
I can access `Register` method in user code, but this access will be useless. Is it a good idea to pass to user code some kind of readonly container?

Reply:
I think user don't need a direct access to Container. For user Container is just Options for the script runner, so some abstraction on top can be exposed